src/App.css

```diff
@@ -1,39 +1,109 @@
+.root {
+  margin: 0;
+  padding: 0;
+}
+
```

Comment:
- [ ] OPTIONAL
The styling of the list on your Homepage does not conform to the alternating dual-color pattern of the provided [design template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). Also two cells should be displayed per row on the mobile screen as seen the [design template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). Therefore kindly ensure that the cells/ card of your list follow the **alternating dual-color pattern**. Each cell of the list item should have surrounding cells having **different (alternating) background color(s)**.

**Your design:**
![image](https://user-images.githubusercontent.com/12745474/194397080-7836ee20-916e-416f-99cb-0b7be7514097.png)


**The original design:**
![image](https://user-images.githubusercontent.com/12745474/142270463-4d16b9d3-7557-430c-80eb-a7c40589a4a6.png)